src/features/student/layouts/StudentLayout/StudentLayout.vue

```diff
@@ -61,7 +61,7 @@ defineExpose({ searchQuery })
   </AvHeader>
 
   <main>
-    <div class="fr-container  fr-mt-3w  fr-mt-md-5w  fr-mb-5w">
+    <div class="fr-container av-container  fr-mt-3w  fr-mt-md-5w  fr-mb-5w">
```

Comment:
```suggestion
    <div class="fr-container  fr-mt-3w  fr-mt-md-5w  fr-mb-5w">
```

J'ai finalement trouvé plus simple, ajoute ça en bas de `src/ui/styles/main.scss`

```vue
.fr-container {
  max-width: 90rem !important;
}
```